docker/README.md

```diff
@@ -23,12 +23,18 @@ The provided [docker-compose.yml](./docker-compose.yml) file will be used to run
 docker-compose stop && docker-compose rm -f alephium && docker-compose pull && docker-compose up -d
 ```
 
-The default config connects your container to the mainnet, and makes the API available to [http://127.0.0.1:12973/docs](http://127.0.0.1:12973/docs):
+The default config connects your container to the mainnet, and makes the API available to [http://127.0.0.1:12973/docs](http://127.0.0.1:12973/docs).
+
+By default an API key with the value of `0000000000000000000000000000000000000000000000000000000000000000` is configured to protect the APIs exposed by
```

Comment:
could you mentioned that API key should be >= 32 alphabetic + number characters

Reply:
Good idea, done :+1: